### DISCUSSION TITLE
Fair usage recommendation

### DISCUSSION THREAD

**Discussion by deadmerc** [tgdrive/teldrive#124] [General]
Hello everyone,

Tool looks great, I check the docs and there was recommendation do not hoard and store Netflix video, but didnt fully understand our borders, which numbers we can put under fair use?

 - 500GB per user / bot, piracy are restricted
 - 100GB per user/ bot, a bit piracy are ok
 - ....?
 
 I just want to understand borders for increase security and stability of my files
 
 Over and out,
 Max

**Comment by bpawnzZ** [comment]
I'm currently around 200gb. Everything is 100% encrypted so I think the piracy point is irrelevant. What do you guys consider 'fair use' per say
